cmd/root.go

```diff
@@ -102,6 +105,19 @@ func runValidate(opts validateOptions) (int, []validate.ValidationResult) {
 
 	fmt.Println("Results:")
 	for _, filename := range opts.filenames {
+		// case stdin:
+		if filename == "-" {
+			fileBytes, err := ioutil.ReadAll(opts.input)
+			if err != nil {
+				fmt.Printf("Error reading stdin: %s\n", err)
+				return 1, totalResults
+			}
+			validationResults := fileValidator.Validate(fileBytes)
```

Comment:
this is duplicated with code starting at line 130. We can extract that into a function or interface for validating a bytes array and reuse it.